CHANGELOG.md

```diff
@@ -62,8 +62,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
   instead of $HOME. #1143
 - Put environment config options in `server_config.json` into a separate
   section named "environment". #1161
-- Automatically register if BlackBox tests are run on a fresh installation.
-  #1180
+- Automatically register if BlackBox tests are run on a fresh installation. #1180
```

Comment:
I've been trying to keep the lines in the changelog <= 80 characters.

Reply:
This was done by black I think. Cause the previous one looked like `1180` is a header.

Reply:
Black shouldn't be formatting markdown files.